### PR TITLE
Init (spring-vault dependency intro)

### PR DESCRIPTION
**Pull request by stingion**

### CODE REVIEW

File: config-vault/src/main/java/io/scalecube/config/vault/VaultConfigSource.java [hunk 1]
```diff
@@ -80,67 +114,67 @@ private void checkVaultStatus() throws VaultException {
...
-            "Unable to build " + VaultConfigSource.class.getSimpleName(), propogateException);
-        throw ThrowableUtil.propagate(propogateException);
-      }
-    }
-
-    public String secretsPath() {
-      return secretsPath;
-    }
-  }
+//  static Builder builder(EnvironmentLoader environmentLoader) {
```
Comment (ronenhamias) [inline config-vault/src/main/java/io/scalecube/config/vault/VaultConfigSource.java]: why this code is muted?

if its not relevant please delete

File: config-vault/src/test/java/io/scalecube/config/vault/VaultConfigSourceTest.java [hunk 2]
```diff
@@ -21,16 +21,19 @@
 import static org.junit.jupiter.api.Assumptions.assumeFalse;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
-import com.bettercloud.vault.EnvironmentLoader;
-import com.bettercloud.vault.SslConfig;
-import com.bettercloud.vault.Vault;
-import com.bettercloud.vault.VaultConfig;
-import com.bettercloud.vault.VaultException;
+//import com.bettercloud.vault.EnvironmentLoader;
```
Comment (ronenhamias) [inline config-vault/src/test/java/io/scalecube/config/vault/VaultConfigSourceTest.java]: lets not leave commented out code

**Comment by dmytro-lazebnyi** [comment]
I have to admit that it was my mistake to propose to use `spring-vault-core` since it brings a lot of overhead to project without any value.

It would solve the problem with integration with the new version of Vault (1.0+), because `vault-java-driver:3.1.0` doesn't support it.

But currently, there is pull request (https://github.com/BetterCloud/vault-java-driver/pull/151) which adds compatibility with Vault 1.0+. This feature not released yet, but pull request already approved and merged (a few days ago) to `master` branch so it is the matter of time when the new release will be available on maven repository.

In light of the above, I suggest to close this pull request and wait for the new release of `vault-java-driver`.

**Comment by dmytro-lazebnyi** [comment]
I have to admit that it was my mistake to propose to use `spring-vault-core` since it brings a lot of overhead to project without any value.

It would solve the problem with integration with the new version of Vault (1.0+), because `vault-java-driver:3.1.0` doesn't support it.

But currently, there is pull request in https://github.com/BetterCloud/vault-java-driver which adds compatibility with Vault 1.0+. This feature not released yet, but pull request already approved and merged (a few days ago) to `master` branch so it is the matter of time when the new release will be available on maven repository.

In light of the above, I suggest to close this pull request and wait for the new release of `vault-java-driver`.

**Pull request closed without merging**